.circleci/config.yml

```diff
@@ -287,7 +278,6 @@ workflows:
                 only: /.*/
   nightly:
     jobs:
-      - 'linter'
```

Comment:
Don't we want to have non-failing linter run for nightly targets (for whole codebase)?

Reply:
good suggestion, I added a daily schedule